src/index.js

```diff
@@ -30,6 +29,7 @@ import Calendar from './components/common/Calendar';
 import MentorMenteeMatching from './components/pages/MentorMenteeMatching/MentorMenteeMatching';
 import Notes from './components/pages/Notes/Notes';
 import NotesForm from './components/pages/Notes/NotesForm';
+import Attendance from './components/pages/Attendance/attendance';
```

Comment:
Why is the attendance component file lowercase? attendance.js should probably be Attendance.js